app/services/fact_parser.rb

```diff
@@ -163,7 +163,10 @@ def get_interfaces
 
   # these interfaces are ignored when parsing interface facts
   def ignored_interfaces
-    /\A(lo(?!cal_area_connection)|usb|vnet|macvtap)/
+    @ignored_interfaces ||= begin
+      prefixes = Setting[:ignore_interfaces_prefixed_with].map { |p| "\\A#{Regexp.escape(p).gsub('\*', '.*')}\\Z" }
```

Comment:
Can you extract this into a helper? This will be useful in discovery. I think we will adopt the same syntax for the same reason. (https://github.com/theforeman/foreman_discovery/pull/200) I think you could put it somewhere in settings, or even to more generic place.